pkg/core/store/badger/workflow.go

```diff
@@ -0,0 +1,171 @@
+package badger
+
+import (
+	"context"
+	"log"
+
+	"github.com/fuseml/fuseml-core/gen/workflow"
+	"github.com/fuseml/fuseml-core/pkg/domain"
+	"github.com/timshannon/badgerhold/v3"
+)
+
+// storableWorkflow holds the workflow and the codesets assigned to it
+type storableWorkflow struct {
+	// Workflow assigned to the codeset
+	Workflow *workflow.Workflow
+	// AssignedCodeset holds codesets assigned to the workflow and its hookID
+	AssignedCodesets []*domain.AssignedCodeset
+}
+
+type WorkflowStore struct {
+	store *badgerhold.Store
+}
+
+func NewWorkflowStore(options badgerhold.Options) *WorkflowStore {
+	store, err := badgerhold.Open(options)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
We should probably return error from `NewWorkflowStore` instead

Reply:
yes!